lib/full-site-editing/full-site-editing.php

```diff
@@ -14,6 +14,15 @@ function gutenberg_is_fse_theme() {
 	return is_readable( get_stylesheet_directory() . '/block-templates/index.html' );
 }
 
+/**
+ * Returns whether the current theme is FSE-enabled or not.
+ *
+ * @return boolean Whether the current theme is FSE-enabled or not.
+ */
+function gutenberg_supports_block_templates() {
+	return gutenberg_is_fse_theme() || ! current_theme_supports( 'disable-full-site-editing' );
```

Comment:
we used to have theme supports like that `disable-full-site-editing` to make them opt-out but I've been told by core devs that we should remove the "disable" here and just call `add_theme_supports` in the Gutenberg code base by default  to make it opt-out. We do that for `widgets-block-editor` for instance.

Reply:
Huh... didn't know about that. 
It does look a lot cleaner... Done. 👍